### ISSUE TITLE
New Additional Learning separate option

### ISSUE THREAD

**Issue by rishav394** [Hariomagr/VIT-DATA-Scraper#2]
Now that Additional Learning is separate I think there should be a self sustaining file for it.

**Comment by Hariomagr** [comment]
Yes, but actually for now i didn't want to mess up my course as maybe there would not come any option to delete the file, so it'll be difficult for me to upload the files. I'll give a try when i will be uploading the additional learning. Meanwhile you can give a try.

**Comment by duttanaman1** [comment]
That would be great help this was done. I will try to submit a PR.
